docs/CMakeLists.txt

```diff
@@ -13,8 +13,6 @@ add_custom_command(
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
   MAIN_DEPENDENCY ${SPHINX_SOURCE}/conf.py
   COMMENT "Generating documentation with Sphinx"
-  # run this command each time. This way we don't have to keep track of each individual rst file
-  BuildDocs
```

Comment:
is this no longer needed?

Reply:
This is apparently incorrect and in the middle of correcting this, I got distracted. It's fixed now

Reply:
Oh, actually it isn't needed. It was meant to be a label to use to build, like make BuildDcos, but we already have one, make docs